docs/templates/tutorials/01-basic-tutorial.md

```diff
@@ -0,0 +1,83 @@
+# 01. Basic Tutorial
+
+In Basic Tutorial, you can learn the following things:
```

Comment:
In this tutorial, you can learn how to:
* Define a Search Space
* Define and optimize an Objective Function